taca/illumina/Runs.py

```diff
@@ -88,6 +98,32 @@ def check_run_status(self):
             for lane in lanes:
                 if self.is_unpooled_lane(lane):
                     self._rename_undet(lane, samples_per_lane)
+            return None
+
+    def _check_demux_log(self, demux_id, demux_log):
+        """
+        This function checks the log files of bcl2fastq
+        Errors or warnings will be captured and email notifications will be sent
+        """
+        with open(demux_log, 'r') as demux_log_file:
+            demux_log_content = demux_log_file.readlines()
+            errors, warnings, error_and_warning_messages = self._extract_errors_and_warnings(demux_id, demux_log_content)
+            return errors, warnings, error_and_warning_messages
+
+    def _extract_errors_and_warnings(self, demux_id, demux_log_content):
```

Comment:
Does this need to be another function? Its called only by `_check_demux_log` and one time too.

Reply:
Function merged into `_check_demux_log`